models/densenet_efficient.py

```diff
@@ -300,16 +300,11 @@ def __init__(self, shared_allocation_1, shared_allocation_2,
         self.momentum = momentum
         self.eps = eps
 
-        # Buffers to store old versions of bn statistics
-        self.prev_running_mean = self.running_mean.new(self.running_mean.size())
-        self.prev_running_var = self.running_var.new(self.running_var.size())
-        self.curr_running_mean = self.running_mean.new(self.running_mean.size())
-        self.curr_running_var = self.running_var.new(self.running_var.size())
```

Comment:
You're right that `curr_running_mean` and `curr_running_var` can be deleted, but I'd prefer to keep `prev_running_mean` rather than using `.clone()`. (See below).